src/Symfony/Component/DependencyInjection/Alias.php

```diff
@@ -94,6 +92,8 @@ public function isPrivate()
      */
     public function setDeprecated(bool $status = true, string $template = null)
     {
+        trigger_deprecation('symfony/dependency-injection', '5.1', 'The "%s()" method is deprecated, use "deprecation()" instead.', __METHOD__);
```

Comment:
let's keep this method and let's remove `setDeprecation()`
we can build a BC layer to change the signature in 6.0